include/cudawrappers/cu.hpp

```diff
@@ -14,6 +14,8 @@
 #include <vector>
 
 #if !defined(__HIP__)
+#include <cuda_runtime.h>
+
 #include <cuda.h>
```

Comment:
Reorder to put them alphabetically, also, the newline is not needed:
```suggestion
#include <cuda.h>
#include <cuda_runtime.h>
```

Reply:
I had them in the wrong order indeed, not sure why pre-commit decided to add a newline instead of switching them around...